Website/admin/Menus/ModuleActions/ModuleActions.css

```diff
@@ -178,16 +178,13 @@ div.actionMenu ul.dnn_mact li.dnn_shared {
   display: inline-block;
   width: 20px;
   height: 100px;
-  background-color: #ffffff;
+  background-color: #000;
 }
 div.actionMenu ul.dnn_mact li.dnn_shared > div {
   display: inline-block;
   width: 20px;
-  height: 20px;
-  writing-mode: tb-rl;
   -webkit-transform: rotate(90deg);
-  -moz-transform: rotate(90deg);
-  -o-transform: rotate(90deg);
+  transform: rotate(90deg);
```

Comment:
The above change (the two lines) are still not producing the expected results. I suggest replacing both with
``text-orientation: sideways;``

Reply:
It's the removal of `writing-mode` that causes it to display correctly for me (in Firefox, Edge, IE, and Chrome).  `text-orientation: sideways;` doesn't have support in IE (http://caniuse.com/#feat=css-text-orientation), and doesn't have any effect in Firefox in my testing.  Let me know if I'm missing something.